actions/distributionsActions.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'isomorphic-fetch';
+import { API_HOST } from '../actions';
+
+const url = '/api/distributions';
+
+const START = 'yasp/distributions/START';
+const DONE = 'yasp/distributions/DONE';
+const ERROR = 'yasp/distributions/ERROR';
+
```

Comment:
we should be consistent with these action names. Can you name them similar to other async requests?